bin/mysociety

```diff
@@ -161,6 +161,7 @@ case $COMMAND in
             shift || die "specify a vhost"
             # Deal with non-sensical commands combined with --all
             if [ "$COMMAND" == "servers" ] || [  "$COMMAND" == "balancers" ]; then die "Using ${COMMAND} with --all makes no sense."; fi
+            if pgrep -f "vhost --all $VHOST" >/dev/null; then die "Someone is currently already running this."; fi
```

Comment:
I think in theory (although rarely in practice) someone could run `vhost (start|stop|update) --all $VHOST`, which this would miss. Do we care given how unlikely this is?

Reply:
Yeah, I thought that was probably really unlikely. But I guess pgrep is a regex, so it could look for them as well easily enough (or even just `'\--all *$VHOST'`?)